rohrpost/mixins.py

```diff
@@ -16,6 +20,12 @@ def on_transaction_commit(func):
         func()
 
 
+def send_to_group(group_name, payload) -> None:
+    if async_to_sync is None:
+        return
+    async_to_sync(channel_layer.send)(group_name, payload)
```

Comment:
Maybe this should be moved to `message.py`?

Reply:
That would be the better location imo, yes.

Reply:
done